src/modules/wreck/lua.d/timeout.lua

```diff
@@ -0,0 +1,40 @@
+--
+-- Register timer on nodeid 0 if kvs `walltime` is set for this job.
+--  Kill job on timeout if reached.
+--
+
+local function start_timer (f, wreck, id, walltime)
+   local to, err = f:timer {
+        timeout = walltime*1000,
+        oneshot = true,
+        handler = function (f, to)
+          wreck:log_msg ("Timeout expired! Killing job")
+          local rc,err = f:sendevent ({signal = 14}, "wreck.%d.kill", id)
```

Comment:
Wouldn't it be better to use signal name (SIGALRM) instead of number?  Also, I expected SIGTERM instead of SIGALRM.